src/screens/Media/Media.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { Suspense, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router';
+
+import useMedia from '#src/hooks/useMedia';
+import ErrorPage from '#src/components/ErrorPage/ErrorPage';
+import { getCustomScreen } from '#src/utils/common';
+import { mediaScreens } from '#src/customScreens';
+import Loading from '#src/screens/Loading/Loading';
+
+const Media = () => {
```

Comment:
I really want to combine Media and Playlist.  They share 90% of the same functionality and having them split forces us to put a bunch of related functionality like getCustomScreen into shared utils, when otherwise it could be non exported, encapsulated logic.

Reply:
We could do a generic `<ScreenRouter type="media | playlist" />` page? Then we can enable the right query and keep all logic in that page component.

Probably also rename `SeriesRouter` to `SeriesRedirect` to clarify the purpose.

Reply:
Those both sound good!